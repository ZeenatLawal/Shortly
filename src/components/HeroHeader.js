import React from 'react';
import working from '../images/illustration-working.svg';

const HeroHeader = () => (
  <section>
    <img src={working} alt="brand" />
    <div>
      <h1>More than just shorter links</h1>
      <p>
        Build your brand&apos;s recognition and get detailed
        insights on how your links are performing.
      </p>
      <button type="button">Get Started</button>
    </div>
  </section>
);

export default HeroHeader;
