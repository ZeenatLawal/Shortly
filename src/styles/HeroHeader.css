.working {
  width: 100%;
}

.hero {
  margin-top: 5rem;
  margin-bottom: 5rem;
  text-align: center;
}

.title {
  margin-top: 2rem;
  font-weight: 700;
  color: var(--darker-violet);
}

@media screen and (min-width: 1024px) {
  .working {
    width: 50%;
    margin-left: 2rem;
  }

  .hero {
    margin-top: 6rem;
    text-align: left;
  }

  .title {
    font-size: 3rem;
  }
}
