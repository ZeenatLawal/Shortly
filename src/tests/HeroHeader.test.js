import renderer from 'react-test-renderer';
import HeroHeader from '../components/HeroHeader';

describe('HeroHeader component', () => {
  test('Snapshot test', () => {
    const header = renderer.create(
      <HeroHeader />,
    )
      .toJSON();
    expect(header).toMatchSnapshot();
  });
});
