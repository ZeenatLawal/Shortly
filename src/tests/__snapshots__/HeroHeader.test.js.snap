// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroHeader component Snapshot test 1`] = `
<section
  className="hero"
>
  <img
    alt="brand"
    src="illustration-working.svg"
  />
  <div>
    <h1
      className="title"
    >
      More than just shorter links
    </h1>
    <p>
      Build your brand's recognition and get detailed insights on how your links are performing.
    </p>
    <button
      type="button"
    >
      Get Started
    </button>
  </div>
</section>
`;
